moveit_core/robot_model/src/planar_joint_model.cpp

```diff
@@ -127,7 +134,11 @@ void moveit::core::PlanarJointModel::getVariableRandomPositionsNearBy(random_num
     values[1] = rng.uniformReal(std::max(bounds[1].min_position_, near[1] - distance),
                                 std::min(bounds[1].max_position_, near[1] + distance));
 
-  double da = angular_distance_weight_ * distance;
+  double da = 0.0;
+  if (fabs(angular_distance_weight_) > std::numeric_limits<double>::epsilon())
+    da = distance / angular_distance_weight_;
+  if (da > boost::math::constants::pi<double>())
+    da = boost::math::constants::pi<double>();
```

Comment:
I misunderstood this [line](https://github.com/ompl/ompl/blob/master/src/ompl/base/src/StateSpace.cpp#L1136). My code is wrong.